nodejs/awsx/apigateway/api.ts

```diff
@@ -507,6 +581,22 @@ function addEventHandlerRouteToSwaggerSpec(
     }
 }
 
+function addRequiredParametersToSwaggerOperation(swaggerOperation: SwaggerOperation, requiredParameters: reqvalidation.Parameter[]) {
+    for (const requiredParam of requiredParameters) {
+        const param = {
+            name: requiredParam.name,
+            in: requiredParam.in,
+            required: true,
+        };
+
+        if (swaggerOperation["parameters"]) {
+            swaggerOperation["parameters"].push(param);
+        } else {
+            swaggerOperation["parameters"] = [param];
+        }
```

Comment:
another pattern we also do is:

```
swaggerOperation["parameters"] = swaggerOperation["parameters"] || [];
swaggerOperation["parameters"].push(param);
```

Reply:
(that's a nit, if you prefer your form, that's fine).

Reply:
thats much nicer! thanks for the suggestion

Reply:
ALSO -- i love nits like these :)